repos/repository_kubebb.yaml

```diff
@@ -3,6 +3,8 @@ kind: Repository
 metadata:
   name: kubebb
   namespace: kubebb-system
+  labels:
+    kubebb.repository.source: official
```

Comment:
pls use same label as kubebb/core

Reply:
the label is `kubebb.repository.source: official` in kubebb/core